Public/Invoke-DockerTag.ps1

```diff
@@ -3,18 +3,33 @@
 function Invoke-DockerTag {
     [CmdletBinding()]
     param (
+        [String]
+        $SourceRegistry = '',
         [Parameter(mandatory=$true)]
         [String]
         $SourceImage,
         [ValidateNotNullOrEmpty()]
         [String]
         $SourceTag = 'latest',
+        [String]
+        $TargetRegistry = '',
         [Parameter(mandatory=$true)]
         [String]
         $TargetImage,
         [ValidateNotNullOrEmpty()]
         [String]
         $TargetTag = 'latest'
     )
-    Invoke-Command "docker tag ${SourceImage}:${SourceTag} ${TargetImage}:${TargetTag}"
+
+    # Add postfix /
+    if (-Not ([String]::IsNullOrWhiteSpace($SourceRegistry))) {
+        $SourceRegistry = Ensure-Postfix -Data $SourceRegistry
```

Comment:
I am against the practice of changing the values of arguments passed into a function. 

I can see why I would make the code shorter here, but I believe it is a bad habit to pick up. 

If you do it when you are passing parameters by reference you will change the values in the calling code, you don't get a copy. This might lead to unexpected behavior in the calling code. So even though it might not create problems here, I suggest:

```suggestion
        $postfixedSourceRegistry = Ensure-Postfix -Data $SourceRegistry
```

to be consistent.

Reply:
Immutable variables FTW! :sun_with_face: